container/docker/Dockerfile.image

```diff
@@ -27,8 +27,8 @@ RUN     apt-get update && \
 		sudo \
 		intel-gpu-tools \
 		libcurl4 python3-yaml pciutils \
-		gstreamer1.0-libav gstreamer1.0-x \
-		libpython3.8 python3 python3-pip && \
```

Comment:
gstreamer1.0-x is required by pose estimation pipeline now

Reply:
yes, don't remove gstreamer1.0-x package